course_dashboard/stats.py

```diff
@@ -175,3 +175,39 @@ def total(self):
     def daily_average(self):
         """Average enrollments per day"""
         return self.total() * 1. / self.day_span()
+
+from certificates.models import GeneratedCertificate, CertificateStatuses
+
+class CertificateStats(object):
+    """Provide certificate stats for a given course."""
+
+    def __init__(self, course_key):
+        self.course_key = course_key
+        self.certificates = GeneratedCertificate.objects.filter(course_id=course_key)
+
+    @property
+    def notpassing(self):
+        """Return the number of failed certificates"""
+        notpassing = 0
+        if not self.certificates:
+            return None
```

Comment:
Pourquoi retourner None et pas 0 ? Après tout il y a bien 0 certificats "not passing".